packages/snaps-utils/src/test-utils/manifest.ts

```diff
@@ -67,44 +67,9 @@ export const MOCK_AUXILIARY_FILE = '{ "foo": "bar" }';
 export const ALTERNATIVE_SNAP_ICON =
   '<svg width="24" height="25" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M17.037 0H6.975C2.605 0 0 2.617 0 6.987v10.05c0 4.37 2.605 6.975 6.975 6.975h10.05c4.37 0 6.975-2.605 6.975-6.976V6.988C24.012 2.617 21.407 0 17.037 0ZM11.49 17.757c0 .36-.18.684-.492.876a.975.975 0 0 1-.54.156 1.11 1.11 0 0 1-.469-.108l-4.202-2.1a1.811 1.811 0 0 1-.985-1.61v-3.973c0-.36.18-.685.493-.877a1.04 1.04 0 0 1 1.008-.048l4.202 2.101a1.8 1.8 0 0 1 .997 1.609v3.974h-.012Zm-.252-6.423L6.723 8.896a1.045 1.045 0 0 1-.528-.924c0-.384.204-.744.528-.924l4.515-2.438a1.631 1.631 0 0 1 1.524 0l4.515 2.438c.324.18.528.528.528.924s-.204.744-.528.924l-4.515 2.438c-.24.132-.504.192-.768.192a1.54 1.54 0 0 1-.756-.192Zm7.972 3.638c0 .684-.385 1.308-.997 1.608l-4.202 2.101c-.144.072-.3.108-.468.108a.975.975 0 0 1-.54-.156 1.017 1.017 0 0 1-.493-.876v-3.974c0-.684.384-1.309.997-1.609l4.202-2.101a1.04 1.04 0 0 1 1.008.048c.313.192.493.516.493.877v3.974Z" fill="#24272A"/></svg>';
 
-// Defined separately to prevent circular dependencies, should match getSnapManifest()
-const SHASUM_MANIFEST = {
-  version: MOCK_SNAP_VERSION,
-  description: MOCK_SNAP_DESCRIPTION,
-  proposedName: MOCK_SNAP_NAME,
-  repository: getDefaultRepository(),
-  source: {
-    shasum: '',
-    location: {
-      npm: {
-        filePath: DEFAULT_SOURCE_PATH,
-        packageName: MOCK_SNAP_NAME,
-        registry: 'https://registry.npmjs.org',
-        iconPath: DEFAULT_ICON_PATH,
-      } as const,
-    },
-  },
-  initialPermissions: MOCK_INITIAL_PERMISSIONS,
-  manifestVersion: '0.1' as const,
-};
-
-export const DEFAULT_SNAP_SHASUM = getSnapChecksum({
-  sourceCode: new VirtualFile({
-    value: DEFAULT_SNAP_BUNDLE,
-    path: DEFAULT_SOURCE_PATH,
-  }),
-  svgIcon: new VirtualFile({
-    value: DEFAULT_SNAP_ICON,
-    path: DEFAULT_ICON_PATH,
-  }),
-  manifest: new VirtualFile({
-    value: JSON.stringify(SHASUM_MANIFEST),
-    result: SHASUM_MANIFEST,
-    path: DEFAULT_MANIFEST_PATH,
-  }),
-  auxiliaryFiles: [],
-  localizationFiles: [],
-});
+// This will need to be recalculated if the checksum inputs change.
+export const DEFAULT_SNAP_SHASUM =
```

Comment:
Can we use top-level await or is that not supported in our environment?

Reply:
It is not supported in our environment as far as I can tell